modules/microshift-configuration.adoc

```diff
@@ -26,11 +26,15 @@ etcd:
 .Verification
 
 . After modifying the `memoryLimitMB` value in `/etc/microshift/config.yaml`, restart {product-title} by running the following command:
++
+[source, terminal]
 ----
 $ sudo systemctl restart microshift
 ----
 
 . Verify the new `memoryLimitMB` value is in use by running the following command:
++
+[source, terminal]
 ----
-$ systemctl status microshift
+$ systemctl show microshift-etcd.scope | grep MemoryHigh
```

Comment:
I learned there's a better way to express this than the version I originally shared.

```suggestion
$ systemctl show --property=MemoryHigh microshift-etcd.scope
```